plugins/woocommerce-blocks/assets/js/blocks/coming-soon/entire-site.scss

```diff
@@ -111,26 +71,6 @@ body:has(.woocommerce-coming-soon-banner) {
 		}
 	}
 
-	.woocommerce-coming-soon-powered-by-woo {
```

Comment:
This is unused since we've already removed the powered-by-woo text from the template.